cpp/include/raft/neighbors/epsilon_neighborhood.cuh

```diff
@@ -98,12 +98,13 @@ void epsUnexpL2SqNeighborhood(bool* adj,
  *                    squared as we compute L2-squared distance in this method)
  */
 template <typename value_t, typename idx_t, typename matrix_idx_t>
-void eps_neighbors_l2sq(raft::resources const& handle,
-                        raft::device_matrix_view<const value_t, matrix_idx_t, row_major> x,
-                        raft::device_matrix_view<const value_t, matrix_idx_t, row_major> y,
-                        raft::device_matrix_view<bool, matrix_idx_t, row_major> adj,
-                        raft::device_vector_view<idx_t, matrix_idx_t> vd,
-                        value_t eps)
+[[deprecated("Use cuVS instead")]] void eps_neighbors_l2sq(
```

Comment:
Has this function been moved to cuvs?

Reply:
Nope, it hasn't :-(. I thought it had but I should have verified. Thank you for pointing this out! Not a high priority so I just removed it from the deprecations for now. I'll add this to our tracker issue after code freeze begins.